src/board/boards/blcu/download.go

```diff
@@ -87,10 +87,12 @@ type fileResponsePayload struct {
 
 func (blcu *BLCU) notifyDownloadFailure() {
 	blcu.trace.Warn().Msg("Download failed")
+	//FIXME: change to toml
 	blcu.sendMessage(os.Getenv("BLCU_DOWNLOAD_TOPIC"), fileResponsePayload{IsSuccess: false, File: nil})
```

Comment:
you can fix this now, it seems I forgot to change these functions, the blcu has a config field which should have the topics

Reply:
Okay. The blcu still has the AddPacket function which will be removed, but I'll do it in the next PR because getting the ACK id is related to some changes that I'm currently working on.